x-pack/legacy/plugins/maps/server/maps_telemetry/maps_usage_collector.test.js

```diff
@@ -4,60 +4,32 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import sinon from 'sinon';
-import { getMockCallWithInternal, getMockKbnServer, getMockTaskFetch } from '../test_utils';
-import { buildCollectorObj } from './maps_usage_collector';
+import { initTelemetryCollection } from './maps_usage_collector';
 
 describe('buildCollectorObj#fetch', () => {
-  let mockKbnServer;
+  let makeUsageCollectorStub;
+  let registerStub;
+  let usageCollection;
 
   beforeEach(() => {
-    mockKbnServer = getMockKbnServer();
+    makeUsageCollectorStub = jest.fn();
+    registerStub = jest.fn();
+    usageCollection = {
+      makeUsageCollector: makeUsageCollectorStub,
+      registerCollector: registerStub,
+    };
   });
 
-  test('can return empty stats', async () => {
```

Comment:
what's the reason all these unit tests are removed?

Reply:
we should add the equivalent tests if possible at all.

Reply:
This was all really just testing our ability to write to and retrieve from the task manager state. Once we got rid of the task manager logic, the tests in `maps_usage_collector.test.js` and `maps_telemetry.test.js` covered the remaining behavior. We could (should) probably at some point augment our `maps_telemetry.test.js` tests, but I think we should tackle that in future telemetry work outside of this bug-fix.